### PR TITLE
Fix bad typehint

### PR DESCRIPTION
**Pull request by norkunas**
# Pull Request

## What does this PR do?
Fixes bad typehint

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

**Review by brunoocasali — COMMENTED** [review]
Thanks, @norkunas you rock! 🤘 

I left a comment because your change raised concern about that! :D

File: src/Contracts/IndexesResults.php [hunk 1]
```diff
@@ -20,7 +22,7 @@ public function __construct(array $params)
     }
 
     /**
-     * @return array<int, array>
+     * @return array<int, Indexes>
```
[the remark this answers is not in the document]
Comment (brunoocasali) [reply src/Contracts/IndexesResults.php:25]: Agreed, do you know how to do that?

[the remark this answers is not in the document]
Comment (norkunas) [reply src/Contracts/IndexesResults.php:28]: I've updated to showcase it.

[the remark this answers is not in the document]
Comment (brunoocasali) [reply src/Contracts/IndexesResults.php:28]: It's perfect for me!

**Pull request merged**